internal/serverinstall/k8s.go

```diff
@@ -293,7 +247,7 @@ func (i *K8sInstaller) Install(
 				grpcPort = spec.Port
 			}
 
-			if spec.Name == "http" {
+			if spec.Name == "https-2" {
```

Comment:
Why the `https-2` change here?

Reply:
The `waypoint-ui` service created by the Helm chart maps port 9702 to the named port `https-2`:

```terminal-output
$ kubectl get svc waypoint-ui -o yaml
// other service yaml
  ports:
  - name: http
    nodePort: 32284
    port: 80
    protocol: TCP
    targetPort: http
  - name: https
    nodePort: 30217
    port: 443
    protocol: TCP
    targetPort: https
  - name: grpc
    nodePort: 30730
    port: 9701
    protocol: TCP
    targetPort: grpc
  - name: https-2
    nodePort: 31750
    port: 9702
    protocol: TCP
    targetPort: https
```